src/index.js

```diff
@@ -1,38 +1,95 @@
+    this.completed = false;
+    this.index = Tasks.taskObj.length + 1;
+  }
+
+  static taskPush(newTask) {
+    Tasks.taskObj.push(newTask);
+  }
+
+  static removeTask(element) {
+    element.parentElement.remove();
+  }
+}
+
+// This function gets call and insert the new task into the HTML
+const fillList = (newTask) => {
+  const content = `<li>
+  <button class="${newTask.completed}"> 
   <i class="fa-regular fa-square"></i>
- </button>
- <div class="view">
-  <label class="label">${x.description}</label>
- </div>
- <button class="${x.index}"> <i class="fa-solid fa-ellipsis-vertical"></i></button>
-</li>`;
-    body.insertAdjacentHTML('beforeend', content);
+  </button>
+  <div class="view">
+  <input type="text" class="description-input" value="${newTask.description}" readonly="">
+  </div>
+  <button class="edit" id="${newTask.description}"> edit</button>
+  <button class="remove"> remove</button>
+  </li>`;
+  body.insertAdjacentHTML('beforeend', content);
+};
+
+// When the user submits a new task we create a new instance of the task obj, and call the methods
+// to add that task to local storage
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const taskInput = document.getElementById('new-task-input');
+  const newTask = new Tasks(taskInput.value);
+  fillList(newTask);
+  Tasks.taskPush(newTask);
+  localStorage.setItem('taskArray', JSON.stringify(Tasks.taskObj));
+  taskInput.value = '';
+});
+
+// Function for emoving the task from the local storage
+const removeTasks = (task) => {
+  const taskDesc = task.querySelector('input').value;
+  const x = JSON.parse(localStorage.getItem('taskArray'));
+  const filterIndex = x.findIndex((x) => x.description === taskDesc);
+  x.splice(filterIndex, 1);
+  x.forEach((item, index) => {
+    item.index = index + 1;
   });
+  localStorage.setItem('taskArray', JSON.stringify(x));
+};
+
+body.addEventListener('click', (e) => {
+  const task = e.target.parentElement;
+  const id = e.target.getAttribute('id');
+  if (e.target.classList.contains('remove')) {
+    removeTasks(task);
+    Tasks.removeTask(e.target);
+  }
+  // task will be edited when first the input field of task is updated and then edit icon is clicked
+  if (e.target.classList.contains('edit')) {
+    const content = document.querySelector('.view');
+    const input = content.querySelector('input');
+    input.removeAttribute('readonly');
+    input.focus();
+    input.addEventListener('blur', (e) => {
+      input.setAttribute('readonly', true);
+      const x = JSON.parse(localStorage.getItem('taskArray'));
+      const filterIndex = x.findIndex((x) => x.description === id);
+      x[filterIndex].description = e.target.value;
+      localStorage.setItem('taskArray', JSON.stringify(x));
+    });
+  }
+});
+
+// Function for displaying the taks from the local storage
+const displayTasks = () => {
+  const tasks = JSON.parse(localStorage.getItem('taskArray'));
+  tasks.forEach((x) => fillList(x));
```

Comment:
- Kindly make sure to create the CRUD features in a new JavaScript file. This is a requirement of the project. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/199876927-922fe424-9b0c-4cbd-a971-85278e7aa88f.png)

- Please ensure that your editing feature is working properly. When you try to edit any task your project is always editing the first one. Kindly fix this. :+1: :smiley: :smiley: